test/token/ERC721/extensions/ERC721Votes.test.js

```diff
@@ -1,184 +1,182 @@
 /* eslint-disable */
 
-const { BN, expectEvent, time } = require('@openzeppelin/test-helpers');
+const { expectEvent, time } = require('@openzeppelin/test-helpers');
 const { expect } = require('chai');
 
-const { getChainId } = require('../../../helpers/chainid');
+const { clock, clockFromReceipt } = require('../../../helpers/time');
 
 const { shouldBehaveLikeVotes } = require('../../../governance/utils/Votes.behavior');
 
-const ERC721Votes = artifacts.require('$ERC721Votes');
+const MODES = {
+  blocknumber: artifacts.require('$ERC721Votes'),
+  // no timestamp mode for ERC20VotesComp yet
```

Comment:
```suggestion
  // no timestamp mode for ERC721Votes yet
```

Do we want to add one ?